infrastructure/restapi/src/main/resources/swagger-ui/patched/swagger-initializer.js

```diff
@@ -0,0 +1,20 @@
+window.onload = function() {
```

Comment:
Its unclear to me what we changed in this file or why it needed to be done. We'll likely need to record that somewhere so that we can handle updates in the future.  And it would be really really good to avoid needing to modify it if we can.

Reply:
We have changed path to json. Originally swagger-ui is a web application in Node.js and there are 4 ways to change json location. In webjars swagger is compiled and the only way to change json location is to change this file if I'm not wrong. People usually copying whole swagger-ui to their resources, changing this one and voila. Instead I've made configuration with only this one changed, as I think it will not be changed from version to version and there will be a test that it was not changed in jar.

Reply:
You could cherry-pick tests touching `/swagger-docs` to clarify it's failing on the current master.